Gems/WhiteBox/Code/Source/EditorWhiteBoxComponent.cpp

```diff
@@ -803,6 +806,27 @@ namespace WhiteBox
         return m_localAabb.value();
     }
 
+    void EditorWhiteBoxComponent::GetVisibleGeometry(const AZ::Aabb& bounds, AzFramework::VisibleGeometryContainer& geometryContainer) const
```

Comment:
This function should be names BuildVisibleGeometry. The current name implies that it is returning built data which is not true.